index.html

```diff
@@ -39,6 +39,197 @@ <h1 class="greeting">I'm Ivan<br>Glad to meet <br>you!</h1>
+
+        <div class="tags">
+          <ul class="tags-ul">
+            <li class="html-tag">html</li>
+            <li class="css-tag">css</li>
+            <li class="javaScript-tag">javaScript</li>
+          </ul>
+        </div>
+
+        <div class="project">
+          <button class="project-btn">View Project</button>
+        </div>
+
+      </div>
+
+      </div>
+    </div>
+
+    <div class="tonic-1">
+
+      <div class="multi-img"></div>
+    <div class="text-box">
+      <div class="tonic-text">
+        <h1 class="tonic-title">Multi-Post <br>Stories</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime2"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime2").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
+
+        <p>A daily selection of privately <br>personalized reads, no accounts or <br>sign-ups required.</p>
+
+        <div class="tags">
+          <ul class="tags-ul">
+            <li class="html-tag">html</li>
+            <li class="css-tag">css</li>
+            <li class="javaScript-tag">javaScript</li>
+          </ul>
+        </div>
+
+        <div class="project">
+          <button class="project-btn">View Project</button>
+        </div>
+
+      </div>
+
+      </div>
+    </div>
+
+    <div class="tonic-1">
+
+      <div class="tonic2-img"></div>
+    <div class="text-box">
+      <div class="tonic-text">
+        <h1 class="tonic-title">Tonic</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime3"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime3").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
+
+        <p>A daily selection of privately <br>personalized reads, no accounts or <br>sign-ups required.</p>
+
+        <div class="tags">
+          <ul class="tags-ul">
+            <li class="html-tag">html</li>
+            <li class="css-tag">css</li>
+            <li class="javaScript-tag">javaScript</li>
+          </ul>
+        </div>
+
+        <div class="project">
+          <button class="project-btn">View Project</button>
+        </div>
+
+      </div>
+
+      </div>
+    </div>
+
+    <div class="tonic-1">
+
+      <div class="multi2-img"></div>
+    <div class="text-box">
+      <div class="tonic-text">
+        <h1 class="tonic-title">Tonic</h1>
+
+        <ul class="info">
+          <li id="client">CANOPY</li>
+          <li id="role">Full stack Dev</li>
+          <li id="date"><span id="datetime4"></span></li>
+          <script>
+            var dt = new Date();
+            document.getElementById("datetime4").innerHTML = dt.getFullYear();
+          </script>
+        </ul>
```

Comment:
-  You did great by using javascript but kindly remove the script pas at this level javascript is not required in the project you will use it later with a better understanding of how it works :+1:

Reply:
Fixed.